bin/upgrade.bash

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../")"
+
+CK8S_STACK="$(basename "$0")"
+export CK8S_STACK
```

Comment:
It is that way because of shellcheck.
```
In bin/upgrade.bash line 5:
export CK8S_STACK="$(basename "$0")"
       ^--------^ SC2155 (warning): Declare and assign separately to avoid masking return values.
```